### PR TITLE
feat: revamp homepage

### PR DESCRIPTION
**Pull request by indraseptama**
### Background

Revamp homepage with new design: https://www.figma.com/file/EYWLwgRYAdbPRfcIHeioXl/Apps-Quran-PLUS?node-id=8077%3A49143&mode=dev

preview:
Screenshot 2023-12-24 at 15 49 30Screenshot 2023-12-24 at 15 49 45Screenshot 2023-12-24 at 15 50 24
Screenshot 2023-12-24 at 15 50 52Screenshot 2023-12-24 at 15 51 18Screenshot 2023-12-24 at 15 49 13
Screenshot 2023-12-24 at 15 54 03Screenshot 2023-12-24 at 15 54 17Screenshot 2023-12-24 at 15 53 44

### Impacted Products

- Login
- Pin
- Homepage
- etc

### How to Test

Please include steps of how you tested this changes

### Pre-Deployment Checklist

- [ ] Run Local OK (Mandatory)
- [ ] Run Dev Build on Real Device

### Test Checklist

- [ ] Unit Testing
- [ ] Other (Please Elaborate)

### Code Review Guidelines

https://oyteam.quip.com/4m8kAafMWWiX/Code-Review-Manifesto

### CODE REVIEW

File: lib/pages/home_page_v2/home_page_v2.dart [hunk 1]
```diff
@@ -89,76 +86,24 @@ class _HomePageV2State extends State<HomePageV2> {
           _launchUrl(state.feedbackUrl);
```
Comment (rhakafebriandi) [inline lib/pages/home_page_v2/home_page_v2.dart:86]: bagian ini juga kayaknya sudah tidak diperlukan lagi ya kak ? 
karena function onFeedbackPress nya sudah tidak ada 
(koreksi aku ya kak kalau ada salah)

**Comment by rhakafebriandi** [comment]
Cuma ada 2 pertanyaan kak dari ku, sisanya aman

File: lib/pages/home_page_v2/home_page_v2.dart [hunk 2]
```diff
@@ -89,76 +86,24 @@ class _HomePageV2State extends State<HomePageV2> {
...
-    ConnectivityStatus connectivityStatus,
-  ) async {
-    if (state.feedbackUrl?.isEmpty ?? true) {
-      notifier.getFeedbackUrl(connectivityStatus);
-    } else {
-      _launchUrl(state.feedbackUrl);
-    }
-  }
-
   Future<void> _launchUrl(String? url) async {
```
[the remark this answers is not in the document]
Comment (indraseptama) [reply lib/pages/home_page_v2/home_page_v2.dart:107]: checking

[the remark this answers is not in the document]
Comment (indraseptama) [reply lib/pages/home_page_v2/home_page_v2.dart]: thank you rhak, jadinya di hapus, bisa dicheck

**Comment by rhakafebriandi** [comment]
approved ya kak

**Review by rhakafebriandi — APPROVED** [review]

**Pull request merged**